### DISCUSSION TITLE
Session Refresh

### DISCUSSION THREAD

**Discussion by badprogrammer42** [headlesshq/headlessmc#203] [Q&A]
Hey, I would like to ask if it is possible to refresh the current mc session after it expired using headlessmc?

**Comment by 3arthqu4ke** [comment]
I cant say for sure, but HeadlessMc already uses the refresh token. So if a session expires in the launcher it probably cant be refreshed.

Or do you mean in-game? There you need to write a mod. Or you use the HMC-Specifics login command to login.
